src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -890,30 +905,38 @@ public sealed partial class LateBinding
         internal LateBinding() { }
         [System.Diagnostics.DebuggerHiddenAttribute]
         [System.Diagnostics.DebuggerStepThroughAttribute]
-        public static void LateCall(object? o, System.Type? objType, string name, object?[]? args, string?[]? paramnames, bool[]? CopyBack) { }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Late binding is dynamic and cannot be statically analyzed, so it may be trimmed")]
```

Comment:
> so it may be trimmed

This could be a little more precise.  "Late binding is dynamic and cannot be statically analyzed. The referenced types and members may be trimmed"